### PR TITLE
feat(EU): smart key battery warning and seat heating new cars

### PR DESCRIPTION
**Pull request by Bubo08**
Bug Fix for Missing data for ccs2 EU #488 

I did also add three additional battery value to the vehicle information:
- SOH state of health in percentage
- Capacity as int value
- Remain (Capacity) as int value

### CODE REVIEW

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 1]
```diff
@@ -415,11 +422,34 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
...
+        flSeatHeatState = get_child_value(state, "Cabin.Seat.Row1.Driver.Climate.State")
+        if flSeatHeatState in [0, 2]:
+            vehicle.front_left_seat_status = False
+        elif flSeatHeatState == 1:
+            vehicle.front_left_seat_status = True
+
+        frSeatHeatState = get_child_value(
+            state, "Cabin.Seat.Row1.Passenger.Climate.State"
+        )
+        if frSeatHeatState in [0, 2]:
```
Comment (cdnninja) [inline hyundai_kia_connect_api/KiaUvoApiEU.py:435]: This appears to be status like all other regions not is_on.  As such should be using the constants. 

https://github.com/Bubo08/hyundai_kia_connect_api/blob/7bf5c6444cd7c0049b065df98685d8280d7f31a6/hyundai_kia_connect_api/const.py#L38-L49

Other states most likely exist for cars as per the above.

**Review by cdnninja — CHANGES_REQUESTED** [review]

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 1 above)
Comment (Bubo08) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: Good point. I did not realize that there were status in the code. I fixed the above. 
I think I will have to check some other attributes.

Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: Sounds good.  What I have found in the past is anything that's a number in the api could be more than true false.  We just don't know it yet. 

We hit that on seats.  For a long time it was 0 and 1. Until we started seeing other options for more advanced cars.  We can cross that bridge when we come to it though.

Comment (Bubo08) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: The code is now more readable. So this is good change in my opinion :-)

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 2]
```diff
@@ -598,9 +613,13 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
         vehicle.air_control_is_on = get_child_value(
             state, "Cabin.HVAC.Row1.Driver.Blower.SpeedLevel"
         )
-        # TODO: vehicle.smart_key_battery_warning_is_on = get_child_value(
-        # TODO:     state, "status.smartKeyBatteryWarning"
-        # TODO: )
+        smart_key_battery_warning_is_on = get_child_value(
```
Comment (ZuinigeRijder) [inline hyundai_kia_connect_api/KiaUvoApiEU.py:616]: This if-else can be better written as:
````
        smart_key_battery_warning_is_on = get_child_value(
            state, "Electronics.FOB.LowBattery"
        ) != 0
````

Comment (Bubo08) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: As python newbie I realized that I even can use the bool function :-)

**Pull request merged**